Source/FieldSolver/FiniteDifferenceSolver/FiniteDifferenceAlgorithms/CKCAlgorithm.H

```diff
@@ -0,0 +1,229 @@
+/* Copyright 2020 Remi Lehe
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+
+#ifndef WARPX_FINITE_DIFFERENCE_ALGORITHM_CKC_H_
+#define WARPX_FINITE_DIFFERENCE_ALGORITHM_CKC_H_
+
+#include <AMReX_REAL.H>
+#include <AMReX_Array4.H>
+#include <AMReX_Gpu.H>
+
+struct CKCAlgorithm {
+
+    static void InitializeStencilCoefficients(
```

Comment:
```suggestion
    static void InitializeStencilCoefficients (
```
The same applies to member functions below, as well as other similar.